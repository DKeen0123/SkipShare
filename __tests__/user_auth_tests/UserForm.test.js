import React from 'react';
import { shallow } from 'enzyme';
import UserForm from '../../user/UserForm';

const userForm = shallow(<UserForm />);

it('renders without crashing', () => {
  expect(userForm).toMatchSnapshot();
});