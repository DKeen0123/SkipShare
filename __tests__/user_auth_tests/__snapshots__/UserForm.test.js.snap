// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Styled(Form)>
  <Styled(Item)
    floatingLabel={true}
  >
    <Styled(Label)>
      Email
    </Styled(Label)>
    <Styled(Input)
      keyboardType="email-address"
      onChangeText={[Function]}
      value=""
    />
  </Styled(Item)>
  <Styled(Item)
    floatingLabel={true}
  >
    <Styled(Label)>
      Password
    </Styled(Label)>
    <Styled(Input)
      onChangeText={[Function]}
      secureTextEntry={true}
    />
  </Styled(Item)>
  <Button
    onPress={[Function]}
  />
</Styled(Form)>
`;
